### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by soerenmartius**
# Changelog
- Add CHANGELOG.md
- Bump TfLint to v0.17.0
- Bump pre-commit to v2.5.1
- Bump Snyk-cli to v1.349.0
- Bump Terraform to v0.12.28

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
```
Comment (mariux) [inline CHANGELOG.md:8]: current changes are missing.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+## [0.0.1] - 2020-05-25
+### Added
+- Initial Release.
+- Add first Makefile implementation with targets for Terraform and pre-commit.
+
+<!-- References -->
+[Unreleased]: https://github.com/mineiros-io/build-tools/compare/v0.5.4...HEAD
+
+[0.5.4]: https://github.com/mineiros-io/build-tools/releases/tag/v0.5.4
```
Comment (mariux) [inline CHANGELOG.md:124]: should be compare not tag.

Comment (soerenmartius) [reply CHANGELOG.md]: resolved in 4a90b4f

File: CHANGELOG.md (hunk 1 above)
Comment (soerenmartius) [reply CHANGELOG.md]: resolved in 4a90b4f

**Pull request merged**